packages/db/prisma/schema.prisma

```diff
@@ -1,65 +1,301 @@
-// This is your Prisma schema file,
-// learn more about it in the docs: https://pris.ly/d/prisma-schema
-
 generator client {
-    provider = "prisma-client-js"
+  provider        = "prisma-client-js"
+  previewFeatures = ["multiSchema"]
 }
 
 datasource db {
-    provider = "postgresql"
-    url      = env("DATABASE_URL")
+  provider  = "postgresql"
+  url       = env("DATABASE_URL")
+  directUrl = env("DIRECT_URL")
+  schemas   = ["auth", "public"]
 }
 
 model Post {
-    id      String @id @default(cuid())
-    title   String
-    content String
-}
-
-// NextAuth.js Models
-// NOTE: When using postgresql, mysql or sqlserver, 
-// uncomment the @db.Text annotations below
-// @see https://next-auth.js.org/schemas/models
-model Account {
-    id                String  @id @default(cuid())
-    userId            String
-    type              String
-    provider          String
-    providerAccountId String
-    refresh_token     String? // @db.Text
-    access_token      String? // @db.Text
-    expires_at        Int?
-    token_type        String?
-    scope             String?
-    id_token          String? // @db.Text
-    session_state     String?
-    user              User    @relation(fields: [userId], references: [id], onDelete: Cascade)
-
-    @@unique([provider, providerAccountId])
-}
-
-model Session {
-    id           String   @id @default(cuid())
-    sessionToken String   @unique
-    userId       String
-    expires      DateTime
-    user         User     @relation(fields: [userId], references: [id], onDelete: Cascade)
-}
-
-model User {
-    id            String    @id @default(cuid())
-    name          String?
-    email         String?   @unique
-    emailVerified DateTime?
-    image         String?
-    accounts      Account[]
-    sessions      Session[]
-}
-
-model VerificationToken {
-    identifier String
-    token      String   @unique
-    expires    DateTime
-
-    @@unique([identifier, token])
+  id        String   @id @default(cuid())
+  title     String
+  content   String
+  published Boolean  @default(false)
+  author    Profile? @relation(fields: [authorId], references: [id])
+  authorId  String?
+
+  @@schema("public")
+}
+
+model Profile {
+  id     String  @id @default(cuid())
+  name   String
+  image  String?
+  email  String?
+  userId String  @db.Uuid
+  user   users   @relation(fields: [userId], references: [id], onDelete: Cascade)
+  posts  Post[]
+
+  @@schema("public")
+}
+
+// 
+// Supabase Generated
+
+model users {
```

Comment:
would be good with a second pair of eyes here - is this the best way to do it?

I tried reading a bit and finding some other examples using supabase+prisma on how to sync the users. Maybe there is a prettier way? Do you need all the auth tables in this schema?

Reply:
I don't have much experience with Prisma, but these are auto generated from the existing tables within auth schema, correct? And if we don't include them here, all of these tables will be deleted, correct?

Reply:
Don't have much experience with Prisma either but it looks similar to our recommendation for our [Prisma integration guide](https://supabase.com/docs/guides/integrations/prisma) I'm not sure that there's a better way at the moment but let me check with the guide authors and get back to you